config/config.yml

```diff
@@ -52,6 +52,7 @@ defaults: &defaults
       format: 'application/xml'
   metadata:
     url: 'https://purl.dlib.indiana.edu/iudl/iucat/%s'
+  purl_redirect_url: 'http://server1.variations2.indiana.edu/cgi-bin/access?%s'
```

Comment:
Can we try an environment check here like our other parameters? Something like:

<%= ENV["PMP_PURL_REDIRECT_URL"] || 'http://server1.variations2.indiana.edu/cgi-bin/access?%s' %>